resources/projects/src/main/java/org/wildfly/quickstart/microprofile/openapi/Fruit.java

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Not sure why we have this src directory under resources/projects. We have jboss-eap and wildfly projects, this src directory seems orphan. Right?

Reply:
Oh right. Most probably it was accidentically copied there

Reply:
Done and thanks!